examples/knative/python/kn-py-cbc-slack/cbc-ini

```diff
@@ -0,0 +1,6 @@
+[customer]
+EncodedCompanyCode = <ENTER YOUR CBC EncodedCompanyCode>
+CompanyCode = <ENTER YOUR CBC CompanyCode>
+BackendServer = dev-prod05.conferdeploy.net
+ProxyServer = 192.168.200.75:3128

```

Comment:
question: is this always needed? can you elaborate on this setting in the README perhaps?